chaosk8s/actions.py

```diff
@@ -1,18 +1,16 @@
 # -*- coding: utf-8 -*-
 import json
 import os.path
-from typing import Union
 
-from chaoslib.exceptions import FailedActivity
-from chaoslib.types import MicroservicesStatus, Secrets
-from logzero import logger
+import yaml
```

Comment:
We favour import sorted alaphabetically :)
So that one should remain the last one in this list.